nav2-usability/scenario_code/lib/deeplink-pathparam/deeplink_pathparam_beamer.dart

```diff
@@ -25,7 +25,8 @@ class BooksApp extends StatelessWidget {
   Widget build(BuildContext context) {
     return MaterialApp.router(
       title: 'Books App',
-      routerDelegate: BeamerRouterDelegate(
+      routerDelegate: BeamerDelegate(
+        notFoundRedirectNamed: '/',
```

Comment:
This was added here because @InMatrix asked me yesterday about redirecting the unknown routes in this specific sample.
I didn't add it to other samples. If I should, I can.